src/frontend/src/components/creacionCuentas/ctaCorriente.vue

```diff
@@ -43,16 +35,34 @@
       </v-container>
     </v-app>
   </div>
+  <footer class="footer">
+    <div class="brand">
+      <p>Marca Registrada &reg;</p>
+    </div>
+  </footer>
```

Comment:
Al momento de revisar la vista, el footer queda mal ubicado. Queda arriba de la lista de sucursales y el botón contratar. Como sugerencia para corregirlo, colocar el footer luego de la línea 35. De ser así, ajustar los márgenes para que el footer no quede tan pegado a lo demás (en mi pantalla sin zoom y resolución 1080x1920, el footer queda muy pegado al container). 

```suggestion
      </v-container>
      <footer class="footer">
        <div class="brand">
          <p>Marca Registrada &reg;</p>
        </div>
      </footer>
    </v-app>
  </div>

```